services/ImageService_test.go

```diff
@@ -1253,4 +1257,40 @@ var _ = ginkgo.Describe("Image Service", func() {
 			})
 		})
 	})
+
+	ginkgo.Describe("List methods with pagination", func() {
```

Comment:
This is not a proper pagination test and you know it 😛. In order to test pagination you need more then one resource in the pool. Please do it properly.